Automattic/wp-calypso
# WP Blaze widget: Turn on V2

## Proposed Changes

* Enable `promote-post/widget-i2` on production
* Enable `promote-post/redesign-i2` on production

## Testing Instructions

* Check the Advertising dashboard and verify that is showing the redesigned version.
* Check the promote widget and make sure version 2 is running. 

## Pre-merge Checklist
